scripts/pull-static

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -e
+
+echo "Pulling fields-normalized.json from cdn"
+python "$(dirname $0)/pull_static.py"
+
+echo "Committing changes"
+git add "$(dirname $(dirname $0))/pystac/static"
+git commit -m "Update fields-normalized.json"
```

Comment:
If there's no changes, this will fail, right? I _think_ we want it to succeed ... it's ok if the file doesn't need to be udpated.

Reply:
Hm, yeah thanks for reminding me that I was a little unsure of that as well :-) ... I'd lean towards leaving the git commit out, and just making it part of the normal incantions you do when releasing (CHANGELOG update, script update, etc, then just commit it all in one or whatever).